benchmarks/naslib/problem.py

```diff
@@ -0,0 +1,155 @@
+import ast
+import os
+import sys
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from kurobako import problem
+
+
+sys.stdout = open(os.devnull, "w")  # Suppress output
+
+
+import naslib.search_spaces as ss  # NOQA
+from naslib.search_spaces.core.graph import Graph  # NOQA
+from naslib.search_spaces.core.query_metrics import Metric  # NOQA
+import naslib.search_spaces.nasbench201.graph as nasbench201_graph  # NOQA
+import naslib.search_spaces.transbench101.graph as transbench101_graph  # NOQA
+from naslib.utils import get_dataset_api  # NOQA
+
+
+class NASLibProblemFactory(problem.ProblemFactory):
+    def __init__(self, search_space: Graph, **config: Dict[str, Any]) -> None:
+        self._search_space = search_space
+        self._config = config
+
+    def specification(self) -> problem.ProblemSpec:
+        if isinstance(self._search_space, ss.TransBench101SearchSpaceMicro):
+            params = [
+                problem.Var(f"x{i}", problem.CategoricalRange(transbench101_graph.OP_NAMES))
+                for i in range(6)
+            ]
+            self._converter = lambda x: [int(z) for z in x]
+        # elif isinstance(self._search_space, ss.NATSBenchSizeSearchSpace):
+        #     params = [problem.Var(f"x{i}", problem.DiscreteRange(1, 8)) for i in range(5)]
+        #     self._converter = lambda x: [z * 8 for z in x]
```

Comment:
The comment outed section should be removed.

Reply:
Thanks, fixed.